msteams-platform/bots/how-to/debug/locally-with-an-ide.md

```diff
@@ -77,17 +77,17 @@ Your bot receives the `conversationUpdate` event as you add the bots to a team,
 
 ## Block a bot in personal chat
 
-Users can choose to block your bot from sending personal chat messages. They may toggle this by right-clicking your bot in the chat channel and choosing **Block bot conversation**. This means, your bots continue to send messages, however, the user doesn't receive the messages.
+Users can choose to block your bot from sending personal chat messages. They may toggle this by right-clicking your bot in the chat channel and choosing **Block bot conversation** that means, your bots continue to send messages, however, the user doesn't receive the messages.
 
 ![Blocking a bot](~/assets/images/bots/botdisable.png)
 
 ## Remove a bot from a team
 
-Users can delete the bot by choosing the trash-can icon on the bots list in their team's view. This only removes the bot from that team's use. Individual users can still interact in personal context. Bots in personal context can't be disabled or removed by users.
+Users can delete the bot by choosing the trash-can icon on the bots list in their team's view that only removes the bot from that team's use. Individual users can still interact in personal context. Bots in personal context can't be disabled or removed by users.
```

Comment:
Undo this @v-habibaumme , @v-psambrani  .
Applies to all sentence that are combined to make them longer.